kube-config-files/secrets-templates/binary-writer-template.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: binary-writer
+type: Opaque
+data:
+  s3.access.key: {{s3.access.key}}
```

Comment:
I think we should name those something like aws.access_key_id and aws.secret_access_key as they are not only for S3, but for all AWS services. And they are read from the etcd keys with this name, as this is the AWS terminology.